images/ipfailover/keepalived/lib/failover-functions.sh

```diff
@@ -46,6 +46,19 @@ function setup_failover() {
     echo "ERROR: Module ip_vs is NOT available."
   fi
 
+  # When the DC supplies an (non null) iptables chain
+  # (OPENSHIFT_HA_IPTABLES_CHAIN) make sure the rule to pass keepalived
+  # multicast (224.0.0.18) is in the table.
+  chain="${OPENSHIFT_HA_IPTABLES_CHAIN:-""}"
+  if [ ! -z ${chain} ]; then
```

Comment:
Unless this needs POSIX compliance, prefer `[[` over `[`. In ether case, use `-n` instead of `! -z`.